src/Microsoft.Azure.EventHubs/Primitives/Fx.cs

```diff
@@ -3,22 +3,18 @@
 
 namespace Microsoft.Azure.EventHubs
 {
+    using System;
     using System.Diagnostics;
 
     static class Fx
     {
-        static ExceptionUtility exceptionUtility;
+        static readonly Lazy<ExceptionUtility> exceptionUtility = new Lazy<ExceptionUtility>(() => new ExceptionUtility());
```

Comment:
>new Lazy<ExceptionUtility>(() => new ExceptionUtility()); [](start = 66, length = 57)

new Lazy<ExceptionUtility>() should be sufficient since calling default constructor.

Reply:
Ohh I wasn't aware of that, I'm reading in the documentation now. 

Thanks a lot every day I'm learning something new :D